src/validator-docs/Rules/Cpf.php

```diff
@@ -4,13 +4,27 @@
 
 namespace geekcom\ValidatorDocs\Rules;
 
+use geekcom\ValidatorDocs\ValidatorFormats;
+
 use function preg_match;
 use function mb_strlen;
 
 final class Cpf extends Sanitization
 {
+    protected function validateFormat($value, $document, $attribute = null)
```

Comment:
Moving this method from the Validator class isn't a good idea. There, it may be used to handle other document types like CNPJ or both.
You may inject it here and use it as you do.

Reply:
Hmmm, good point! What do you think of isolating the method and changing the format validation to something like `validateCpfFormat(string $value)`? In this case, we may opt to move it to a similar context.

Please, apply the parameter's type hint to avoid errors in the future.

@geekcom any point more to add to this discussion?